includes/validation/class-amp-validation-manager.php

```diff
@@ -469,6 +469,14 @@ public static function add_admin_bar_menu_items( $wp_admin_bar ) {
 			'href'   => esc_url( is_amp_endpoint() ? $non_amp_url : $amp_url ),
 		];
 
+		// Construct admin bar item to link to AMP version or non-AMP version.
+		$paired_browsing_item = [
+			'parent' => 'amp',
+			'id'     => 'amp-paired-browsing',
+			'title'  => esc_html__( 'Start paired browsing', 'amp' ),
+			'href'   => remove_query_arg( amp_get_slug(), add_query_arg( AMP_Theme_Support::PAIRED_BROWSING_QUERY_VAR, '1' ) ),
+		];
+
```

Comment:
Should this be moved into the condition below? No need to create this variable if it's not going to be used.